langstream-agents/langstream-agent-grpc/src/test/java/ai/langstream/agents/grpc/GrpcAgentSourceTest.java

```diff
@@ -251,6 +251,11 @@ public String getGlobalAgentId() {
             return null;
         }
 
+        @Override
+        public String getAgentId() {
```

Comment:
how does this work for the "composite agent".
I think this is wrong.
The agent can use this.getAgentId()
we should not provide this method in the "AgentContext", that is shared among all the agents in a composite pipeline